src/features/filter.ts

```diff
@@ -1,10 +1,55 @@
-export const actions = { /* put action creators here */};
+import { Status } from '../types/Status';
```

Comment:
Consider using a more descriptive import name for 'Status' like 'TodoStatus' to make the code more readable 📖